### ISSUE TITLE
Cant Record (Failed to initialize codec) 

### ISSUE THREAD

**Issue by Porol** [jackun/TestAMFVFW#9]
Hello,
Programs see codec. I can change the settings. But in recording errors

format AVI:
MSI AB - create an empty file records
Bandicam - alert error: Failed to initialize codec.

format MKV:
MSI AB - The video with a black screen and sound

LOG MSI AB:
Compression query: 24 0 1280x1024
CompressQuery OK
Compression query: 24 0 1280x1024
CompressQuery OK
CompressEnd

LOG Bandicam:
Compression query: 12 32315659 1280x1024
CompressQuery OK
Compression query: 12 32315659 1280x1024
CompressQuery OK
CompressEnd
Compression query: 24 0 1280x1024
CompressQuery OK
Compression query: 24 0 1280x1024
CompressQuery OK
CompressEnd

Monitor: 1280x1024(5:4)
MSI R9 270X (GCN 1.0)
Windows 7 x64
All software/driver latest versions
TestAMFVFW - Build date: Dec 11 2015
I installed "Crimson Edition 16.1 Hotfix", 2013 runtimes (x86 and x64)

P.S. - OBS VCE(AMF) - works well, no problem.
Thanks.

**Comment by CharliBrwn** [comment]
Download the Microsoft Media Feature Pack for your operating system. (x86 and the x64 version) Saw this solution on the Dxtory forum, could work for you.

**Comment by Porol** [comment]
"Microsoft Media Feature Pack" need for Windows 7 Ultimate N or KN. I have Windows 7 Ultimate(full version)

I found the reason. I inattentively read Readme, namely:

> Click  Download ZIP  and unpack somewhere permanent. Codec needs to find the AMF DLLs from the "Runtimes" folder.

I installed the codec. Then delete the "TestAMFVFW-master" folder(In this folder is the "Runtimes" folder)
And the key in the registry referred to is not an existing folder: HKCU\Software\TestAMFVFW\InstallPath

Now that is properly installed. All works.

note: if both the codecs "OpenEncodeVFW" and "TestAMFVFW" installed. After uninstalled of the "Open Encoder VFW", then the programs can not find a codec "TestAMFVFW".
